source/language/subroutines.rst

```diff
@@ -5,10 +5,11 @@ Subroutines are declared using the statement ``def name(parameters) qargs -> out
 Zero or more quantum bits
 and registers are passed to the subroutine by reference or name in ``qargs``.
 Classical types are passed by value in ``parameters``. The parentheses may be omitted if no
-``parameters`` are passed. The subroutines return up to
-one classical type. All arguments are declared together with their type,
-for example ``qubit: ancilla`` would define a quantum bit argument named ``ancilla``. Qubit
-declarations are not allowed within subroutines as they are global. A subroutine
+``parameters`` are passed. The subroutines return up to one classical type, signified by the
```

Comment:
```suggestion
``parameters`` are passed. The subroutines return up to one value of a classical type, signified by the
```

Reply:
Addressed in https://github.com/Qiskit/openqasm/pull/140/commits/f5ea050b6c7c47ec9d236139689023568aebcc99